resources/lib/bom/bom_radar.py

```
@@ -227,41 +245,24 @@ def build_images(radar_code, path, loop_path):
-    # connected, so let's get the list
-    try:
-        # BOM FTP still, in 2021, does not support the nicer mdst() operation
-        files = ftp.nlst()
-        files.sort(reverse=True)
-    except ftplib.error_perm as resp:
-        if str(resp) == "550 No files found":
-            Logger.error("No files in BOM ftp directory!")
-        else:
-            Logger.error("Something wrong in the ftp bit of radar images")
-            Logger.error(str(resp))
+            with ftplib.FTP("ftp.bom.gov.au", timeout=15) as ftp:
```

Ruff (0.13.1)

FTP-related functions are being called. FTP is considered insecure. Use SSH/SFTP/SCP or some other encrypted protocol.

(S321)